tests/tests/legacy_kustomizations/kfserving-install/test_data/expected/~g_v1_configmap_inferenceservice-config.yaml

```diff
@@ -1,100 +1,100 @@
 apiVersion: v1
 data:
-  credentials: |-
+  credentials: |
```

Comment:
I forgot what's the difference between `|-` and `|`, is it ok to move dash?

Reply:
This is generated by the test tool indeed.

The difference is `|-` removes the newline from the last line of the string. In the context of changes here, it is ok.